app/server.go

```diff
@@ -1,24 +1,38 @@
 package app
 
 import (
-	"chat/repository"
+	"chat/messaging/delivery/http"
+	"chat/messaging/repository"
+	"chat/messaging/usecase"
+	"github.com/gin-gonic/gin"
 	"github.com/gocql/gocql"
 	"log"
 	"os"
+	"time"
 )
 
+func server(mh *http.MessageHandler) *gin.Engine {
+	router := gin.Default()
+	mapUrls(router, mh)
+	return router
+}
 
 // Start runs the server
 func Start() {
-
 	cluster := gocql.NewCluster(os.Getenv("CASSANDRA_HOST"))
 	cluster.Keyspace = os.Getenv("CASSANDRA_CHAT_KEYSPACE")
 
 	session, err := cluster.CreateSession()
-	if  err != nil {
+	if err != nil {
 		log.Fatalln(err)
 	}
 	log.Printf("Connected Cassandra database OK")
 
-	repository.NewChatRepository(session)
+	mr := repository.NewChatRepository(session)
```

Comment:
I personally think their meaning should be obvious from the context and the short name does fit the style guide as seen [here](http://wordaligned.org/articles/go-for-short-variable-names). However, I will take this advice to change the parameter name in the Server function to be more descriptive. What do you think?

Reply:
Since the variables were only used once here it's fine. In general, I find it difficult to follow with really short names, but the short names within an easy to find scope like in this case is understandable. It's also a personal preference for me to have more descriptive names, comments in the article you linked show pple who have different opinions like you and me. In general, if the variable names pile up in a larger file, then I think it's better to make the names more descriptive, just so it's readable for others, it's not always obvious nor easy to remember short var name meanings for others who didn't write the code. This is just a heads up for the future, you don't have to change it this time